substrate/runtime/balances/src/lib.rs

```diff
@@ -140,21 +134,17 @@ decl_module! {
 	}
 }
 
-/// An event in this module.
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
-#[derive(Encode, Decode, PartialEq, Eq, Clone)]
-pub enum RawEvent<AccountId, AccountIndex, Balance> {
-	/// A new account was created.
-	NewAccount(AccountId, AccountIndex, NewAccountOutcome),
-	/// An account was reaped.
-	ReapedAccount(AccountId),
-	/// Transfer succeeded (from, to, value, fees).
-	Transfer(AccountId, AccountId, Balance, Balance),
-}
-
-impl<A, I, B> From<RawEvent<A, I, B>> for () {
-	fn from(_: RawEvent<A, I, B>) -> () { () }
-}
+impl_event!(
+	pub enum Event<T> with RawEvent<AccountId, AccountIndex, Balance>
+		where <T as system::Trait>::AccountId, <T as Trait>::AccountIndex, <T as Trait>::Balance {
```

Comment:
to avoid the problem of the indentation change not reflecting the block start, we move the trailing curly to the previous indent level and give it its own line in these circumstances., so this would become:

```rust
pub enum Event<T> with RawEvent<AccountId, AccountIndex, Balance>
	where <T as system::Trait>::AccountId, <T as Trait>::AccountIndex, <T as Trait>::Balance
{
	/// A new account was created.
```

if the `where` clause becomes too big then this works:

```rust
pub enum Event<T> with RawEvent<AccountId, AccountIndex, Balance> where
	<T as system::Trait>::AccountId,
	<T as Trait>::AccountIndex,
	<T as Trait>::Balance
{
	/// A new account was created.
```